recipes/default.rb

```diff
@@ -27,6 +27,29 @@ class Chef::Resource
 
 include_recipe 'erlang'
 
+version = node['rabbitmq']['version']
+
+default_deb_package_name = "rabbitmq-server_#{version}-1_all.deb"
+default_deb_package_url = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{version}/"
```

Comment:
It's a good opportunity to switch this URL away fro m`rabbitmq.com/releases`, say, to [GitHub releases](https://github.com/rabbitmq/rabbitmq-server/releases). GitHub offers better availability, a CDN and there are not plans to drop it as an artifact distribution mechanism, unlike `rabbitmq.com/releases`.